browser/resources/settings/brave_system_page/brave_vpn_page.html

```diff
@@ -0,0 +1,29 @@
+<if expr="enable_brave_vpn">
+<style include="settings-shared cr-shared-style iron-flex">
+  .title {
+    font-size: 14px;
+    font-weight: 600;
+    height: 68px;
+  }
+</style>
+
+<template is="dom-if" if="[[showVpnPage_()]]">
+  <div class="cr-row title">$i18n{vpnPageTitle}</div>
+  <settings-vpn-page prefs="{{prefs}}">
+
+    <settings-toggle-button id="toggleWireguardButton"
+        pref="{{prefs.brave.brave_vpn.wireguard_enabled}}"
+        label="$i18n{useWireguardLabel}"
+        sub-label="[[toggleWireguardSubLabel_]]"
+        disabled="[[braveVpnConnected_]]"
+        on-change="onChange_">
```

Comment:
Should we render this only when it's os win?

Reply:
done

Reply:
this seems to be causing a problem on macOS